### PR TITLE
Fix UG and Sort Feature bugs

### PR DESCRIPTION
**Pull request by ReubenChay**
Fixed some minor bugs reported during PE-D

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -237,20 +245,32 @@ Format: `sort KEYWORD`
...
-* `name` (by alphabetical order)
-* `email` (clients using the same email platforms are grouped together, thereafter, alphabetical order is used within each group)
-* `phone` (by numerical order of phone number)
-* `birthday` (from oldest to youngest)
-* `income` (from highest to lowest income)
-* `risk appetite` (from highest to lowest)
+
+Below is an example of what you will expect to see when you call `sort email`:
+
+:exclamation: Add picture here
```
Comment (marcusczh) [inline docs/UserGuide.md:271]: Just to check, is this placeholder intended?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -237,20 +245,32 @@ Format: `sort KEYWORD`
...
+
+| KEYWORD          | Function                                                                                                             |
+|:-----------------|:---------------------------------------------------------------------------------------------------------------------|
+| `default`        | Based on when a client was added, from oldest to newest                                                              |
+| `name`           | By alphabetical order ie. "a" to "z"                                                                                 |
+| `email`          | Clients using the same email platform are grouped together, thereafter, alphabetical order is used within each group |
+| `phone`          | By numerical order of the phone number                                                                               |
+| `birthday`       | From oldest to youngest   |                                                                                            
+| `income`         | From highest to lowest income |
+| `risk appetite`  | From highest to lowest ie. "H" to "M" to "L" |
```
Comment (marcusczh) [inline docs/UserGuide.md:261]: Great use of tables to make the the document look neater!

File: docs/UserGuide.md (hunk 1 above)
Comment (ReubenChay) [reply docs/UserGuide.md:271]: yes sir i haven't screenshot yet

**Comment by marcusczh** [comment]
LGTM!

**Pull request merged**